CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.3.4] - 2022-09-01
+### Changed
+
+- Created configuration to OAuth server port.
```

Comment:
I think we could update this description to something like: _Created setting to control the port number on which a server will be started on the local machine to proceed with the OAuth2 authorization flow_

Reply:
It is a better description to the changelog too. I'm going to fix it too.